src/pages/community/team.mdx

```diff
@@ -12,4 +14,13 @@ import TeamCard from "../../components/TeamCard"
     Fan of Anime, Sci Fi Movies and Shows
   </TeamCard>
   
+  // LATHRYX CARD 
+  <TeamCard 
+    className={'col col--6 margin-bottom--lg'} 
+    name="Lathryx" 
+    imageUrl="../../img/lathryx-avatar-square.jpg"
+    githubUrl="https://github.com/lathryx" 
+   > 
+   Front-end? Back-end? I can't decide! Love working with React, and I have a good grasp of HTML, CSS, and Javascript. Can't forget Python! <3 
```

Comment:
Try removing <3 as it can give an error =D

Reply:
Okay. :)

Reply:
It has been fixed, I switched it out with the heart emoji. ❤